lib/matplotlib/mlab.py

```diff
@@ -1274,6 +1274,9 @@ def specgram(x, NFFT=None, Fs=None, detrend=None, window=None,
     """
     if noverlap is None:
         noverlap = 128
+    if len(x) <= (256 if NFFT is None else NFFT):  # see _spectral_helper()
```

Comment:
same here, I would encode the default of 256 so that it can not get out of sync with the down stream functions.